src/DataTable/DataTable.jsx

```diff
@@ -14,38 +14,46 @@ export class DataTable extends Component {
   }
   
   static defaultProps = {
-    autoResize: false
```

Comment:
Any prop that isn't required should have a default:

```javascript
static defaultProps = {
  autoResize: true,
  checkboxColumn: false,
  resizeDelay: 100,
  stripeRows: true
}
```

Notice that I added resizeDelay so that it can be configured when we use debounce.

Also note that your propTypes will need to change. you will need to add `resizeDelay: PropTypes.number`